data/core/view.lua

```diff
@@ -66,31 +72,59 @@ function View:get_scrollbar_rect()
 end
 
 
+function View:get_h_scrollbar_rect()
+  local sz = self:get_h_scrollable_size()
+  if sz <= self.size.x or sz == math.huge then
+    return 0, 0, 0, 0
+  end
+  local w = math.max(20, self.size.x * self.size.x / sz)
+  return
+    self.position.x + self.scroll.x * (self.size.x - w) / (sz - self.size.x),
+    self.position.y + self.size.y - style.scrollbar_size,
+    w,
+    style.scrollbar_size
+end
+
+
 function View:scrollbar_overlaps_point(x, y)
   local sx, sy, sw, sh = self:get_scrollbar_rect()
-  return x >= sx - sw * 3 and x < sx + sw and y >= sy and y < sy + sh
+  return x >= sx - sw * 3 and x < sx + sw and y >= sy and y <= sy + sh
+end
+
+
+function View:h_scrollbar_overlaps_point(x, y)
+  local sx, sy, sw, sh = self:get_h_scrollbar_rect()
+  return x >= sx and x <= sx + sw and y > sy - sh * 3 and y <= sy + sh
```

Comment:
Like comment above: there is a reason why you are not using y < sy + sh ? I don't mean it is wrong, just asking.

Reply:
Same reason of the vertical one. A pixel at the bottom of the scrollbar not accounted for.

Reply:
Fine.